esp-hal/src/lcd_cam/mod.rs

```diff
@@ -59,6 +59,17 @@ pub enum ByteOrder {
     Inverted = 1,
 }
 
+/// LCD_CAM possible clock sources to drive the peripheral
+#[derive(Debug, Clone, Copy, PartialEq, Default)]
+#[cfg_attr(feature = "defmt", derive(defmt::Format))]
+#[repr(u32)]
+pub enum LcdClockSource {
+    #[default]
```

Comment:
Why is this the default? I'd expect the default to be no clock, so `Option<LcdClockSource>` with a default of `None`.